### DISCUSSION TITLE
How can I cross compile Fairy-Stockfish for linux on windows?

### DISCUSSION THREAD

**Discussion by ginkgo20** [fairy-stockfish/Fairy-Stockfish#656] [Q&A] [answered]
I couldn't find info about cross-compiling Linux binaries in Windows, most questions are about the opposite situation.

Has someone any experience to share? **I prefer to use MinGW.**

**Comment by ianfab** [comment]
Since the toolchain is the same as for official SF, I would just refer to whatever will be discussed in https://github.com/official-stockfish/Stockfish/discussions/4567.

**Comment by gbtami** [comment]
Why not try to use WSL? https://learn.microsoft.com/en-us/windows/wsl/

**Reply by gbtami** [reply]
WSL enables you to use Linux (compile) tool set on Windows without installing Linux. It will produce exactly the same binaries you can create on Linux. But if you want to do it on a harder way, good luck :)

**Reply by gbtami** [reply, marked as answer]
Btw MinGW is for cross compiling fo Windows. Not the opposite.

**Comment by ginkgo20** [comment]
There is no way for cross compiling via MinGW.
